package.json

```diff
@@ -12,6 +12,7 @@
     "co-body": "4.0.0",
     "co-pg": "1.3.1",
     "config": "1.19.0",
+    "ejs": "2.4.1",
```

Comment:
It would be nice to use backtick templating only (our package.json is really fat already)

```js
# template.js

return ({ name, anotherVar }) => `
    <h1>Hello ${name}<h1>
    <pre>Here is another var: ${anotherVar}</pre>
`;

# usage
const email = require('template.js')({
    name: "The Name",
    anotherVar: "The Var",
})
```